docs/contributors-guide.md

```diff
@@ -1,34 +1,30 @@
-# **Contributor Orientation Guide**
+# **Contributing**
 
-We welcome contributions from anyone, in particular, students (see [here](https://teammatesv4.appspot.com/about.jsp) for a list of our contributors). One of the main objectives of TEAMMATES is to help students get experience in a OSS production environment. Here are some information that might be useful to would-be contributors.
+TEAMMATES is a community that lives on its volunteers, most of whom are [students](https://teammatesv4.appspot.com/about.jsp). One of our main objectives is to help students get experience in an OSS production environment.
+
+## Advice for new contributors
+Contributing to an OSS project requires you to figure out things on your own when you can, and seek help from the right resources (Google, StackOverflow, [troubleshooting guide](troubleshooting-guide.md), [issue tracker](https://github.com/TEAMMATES/teammates/issues), etc.). To become a TEAMMATES contributor, you need to start honing those skills.
+
+To help you with that, we have created a series of easy tasks you can should start off with.
+
+**Table of Contents**
 
 * [Knowledge required](#knowledge-required)
 * [Getting started](#getting-started)
 * [Orientation task list](#orientation-task-list)
+* [Roles](#roles)
 * [Contacting us](#contacting-us)
 
-## Knowledge required
+## Learning the basics
```

Comment:
I think this is better as the knowledge is not really required - the content suggests you pick them up as you go along.